src/asciidoc/docs/customizations/video.adoc

```diff
@@ -1,15 +1,18 @@
 = Video
 :page-slug: video
+:page-description: Standard VCC for uploading video files with automatic transcoding and formatting.
 
-* {blank}
+* Users can
 //tag::description[]
-Uploaded video is automatically transcoded to be appropriate for playback across devices.
+upload video files.
 //end::description[]
+* Uploaded videos are automatically compressed, transcoded to .mp4 format, and cropped to fit 16:9 portrait for optimal playback across devices.
 * Pose detection data can be generated by passing an optional `typeOption`.
 When specified, pose data is available after upload by appending `.poses` to the returned URL.
 For example, `\https://objects.koji-cdn.com/project-id/my-video.mp4.poses`.
```

Comment:
I'm curious about the format of the poses file, but I don't think it's essential to have that info now. Personally, I haven't used this feature before.

Reply:
Thanks! Added some info to clarify this point!